DependencyInjection/Configuration.php

```diff
@@ -195,6 +195,26 @@ public function getConfigTreeBuilder()
                     ->end()
                 ->end()
             ->end()
+            ->scalarNode('twig_mode')
```

Comment:
```suggestion
            ->enumNode('twig_mode')
                ->values('none', 'lazy', 'legacy')
```

We could use `enumNode` and there is no need of the first `validate` call.